### ISSUE TITLE
Job Launch prompt window closes when trying to launch a job

### ISSUE THREAD

**Issue by supremedacos** [ansible/awx#10193]
##### ISSUE TYPE
 - Bug Report

##### ENVIRONMENT
* AWX version: 19.1.0
* AWX install method: k8s docker on linux via awx-operator
* Ansible version:  2.10.8
* Operating System: CentOS Linux release 8.3.2011
* Web Browser: Chrome & Safari

##### STEPS TO REPRODUCE

1. From main web UI page select `Templates`
2. In the search field, search for a template with a name that exists
  The URL will change to something like this - `https://awx.../#/templates?template.name__icontains=sleep`
3. Click the launch rocket button on the right for a template that has an Inventory `Prompt on Launch` checked
4. When the new prompt window is displayed, click on the single `>` button to navigate to the next page of inventories
  The URL will change to something like this - `https://awx.../#/templates?inventory.page=2`
5. Wait 5 seconds

##### EXPECTED RESULTS

The prompt window does not disappear and either I get the final screen to launch the job or more prompt screens.

##### ACTUAL RESULTS

The prompt window disappears.

##### ADDITIONAL INFORMATION

awx-web logs when attempting to launch a job
```
2021-05-13 17:00:29,112 INFO     [bb849c73529441499d799a6435f6652f] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/unified_job_templates/'>, response_time: 0.401s
[pid: 29|app: 0|req: 823/2886] 10.43.237.14 () {74 vars in 3359 bytes} [Thu May 13 17:00:28 2021] OPTIONS /api/v2/unified_job_templates/ => generated 9598 bytes in 403 msecs (HTTP/1.1 200) 14 headers in 571 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:29 +0000] "OPTIONS /api/v2/unified_job_templates/ HTTP/1.1" 200 9598 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.10"
2021-05-13 17:00:29,155 INFO     [47c5d4d8111747d4b617281c2bb775ee] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/workflow_job_templates/'>, response_time: 0.445s
[pid: 31|app: 0|req: 434/2887] 10.43.237.14 () {74 vars in 3361 bytes} [Thu May 13 17:00:28 2021] OPTIONS /api/v2/workflow_job_templates/ => generated 15282 bytes in 448 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:29 +0000] "OPTIONS /api/v2/workflow_job_templates/ HTTP/1.1" 200 15282 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.13"
2021-05-13 17:00:29,272 INFO     [704e9b6352d2470b94a13e753808cb33] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/job_templates/'>, response_time: 0.564s
[pid: 27|app: 0|req: 415/2888] 10.43.237.14 () {74 vars in 3343 bytes} [Thu May 13 17:00:28 2021] OPTIONS /api/v2/job_templates/ => generated 23522 bytes in 566 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:29 +0000] "OPTIONS /api/v2/job_templates/ HTTP/1.1" 200 23522 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.10"
2021-05-13 17:00:29,569 INFO     [d473fa5a90854d98a9242966c2a299e3] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_job_templates/?name__icontains=sleep&order_by=name&page=1&page_size=20&type=job_template%2Cworkflow_job_template'>, response_time: 0.861s
[pid: 30|app: 0|req: 661/2889] 10.43.237.14 () {72 vars in 3505 bytes} [Thu May 13 17:00:28 2021] GET /api/v2/unified_job_templates/?name__icontains=sleep&order_by=name&page=1&page_size=20&type=job_template%2Cworkflow_job_template => generated 4881 bytes in 865 msecs (HTTP/1.1 200) 14 headers in 571 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:29 +0000] "GET /api/v2/unified_job_templates/?name__icontains=sleep&order_by=name&page=1&page_size=20&type=job_template%2Cworkflow_job_template HTTP/1.1" 200 4881 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.13"
2021-05-13 17:00:31,403 INFO     [259a4fd8749844c082e0603c85bc2f76] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/job_templates/156/survey_spec/'>, response_time: 0.546s
[pid: 30|app: 0|req: 662/2890] 10.43.237.14 () {72 vars in 3326 bytes} [Thu May 13 17:00:30 2021] GET /api/v2/job_templates/156/survey_spec/ => generated 2 bytes in 547 msecs (HTTP/1.1 200) 14 headers in 582 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:31 +0000] "GET /api/v2/job_templates/156/survey_spec/ HTTP/1.1" 200 2 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.10"
2021-05-13 17:00:31,445 INFO     [8e354a61361444d3bc3b1b4b6b233d99] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/job_templates/156/launch/'>, response_time: 0.588s
[pid: 28|app: 0|req: 557/2891] 10.43.237.14 () {72 vars in 3316 bytes} [Thu May 13 17:00:30 2021] GET /api/v2/job_templates/156/launch/ => generated 835 bytes in 591 msecs (HTTP/1.1 200) 14 headers in 576 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:31 +0000] "GET /api/v2/job_templates/156/launch/ HTTP/1.1" 200 835 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.13"
2021-05-13 17:00:31,938 INFO     [6749bb1a2c424714936b0b7841bd7ba6] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/inventories/'>, response_time: 0.412s
[pid: 30|app: 0|req: 663/2892] 10.43.237.14 () {74 vars in 3339 bytes} [Thu May 13 17:00:31 2021] OPTIONS /api/v2/inventories/ => generated 14034 bytes in 414 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:31 +0000] "OPTIONS /api/v2/inventories/ HTTP/1.1" 200 14034 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.10"
2021-05-13 17:00:32,145 INFO     [a7dae707be0b44d58a6084d6fefc17e9] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/inventories/?order_by=name&page=1&page_size=5'>, response_time: 0.619s
[pid: 29|app: 0|req: 824/2893] 10.43.237.14 () {72 vars in 3355 bytes} [Thu May 13 17:00:31 2021] GET /api/v2/inventories/?order_by=name&page=1&page_size=5 => generated 11993 bytes in 621 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:32 +0000] "GET /api/v2/inventories/?order_by=name&page=1&page_size=5 HTTP/1.1" 200 11993 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.13"
2021-05-13 17:00:34,421 INFO     [c1e3a98e7b73409d96bdf0c831460201] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/inventories/'>, response_time: 0.431s
10.43.237.14 - - [13/May/2021:17:00:34 +0000] "OPTIONS /api/v2/inventories/ HTTP/1.1" 200 14034 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.10"
[pid: 27|app: 0|req: 416/2894] 10.43.237.14 () {74 vars in 3339 bytes} [Thu May 13 17:00:33 2021] OPTIONS /api/v2/inventories/ => generated 14034 bytes in 433 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
2021-05-13 17:00:34,462 INFO     [94d75cee670044df9ca9ae5948e28300] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/workflow_job_templates/'>, response_time: 0.471s
[pid: 31|app: 0|req: 435/2895] 10.43.237.14 () {74 vars in 3361 bytes} [Thu May 13 17:00:33 2021] OPTIONS /api/v2/workflow_job_templates/ => generated 15282 bytes in 473 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:34 +0000] "OPTIONS /api/v2/workflow_job_templates/ HTTP/1.1" 200 15282 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.15"
2021-05-13 17:00:34,534 INFO     [a84afe9288004f56b10796d0a41d63c9] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/job_templates/'>, response_time: 0.542s
[pid: 30|app: 0|req: 664/2896] 10.43.237.14 () {74 vars in 3343 bytes} [Thu May 13 17:00:33 2021] OPTIONS /api/v2/job_templates/ => generated 23522 bytes in 546 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:34 +0000] "OPTIONS /api/v2/job_templates/ HTTP/1.1" 200 23522 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.10"
2021-05-13 17:00:34,593 INFO     [3a519fcbf3904d75b8f8871d8b8167f1] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/inventories/?order_by=name&page=2&page_size=5'>, response_time: 0.604s
[pid: 28|app: 0|req: 558/2897] 10.43.237.14 () {72 vars in 3355 bytes} [Thu May 13 17:00:33 2021] GET /api/v2/inventories/?order_by=name&page=2&page_size=5 => generated 12114 bytes in 607 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:34 +0000] "GET /api/v2/inventories/?order_by=name&page=2&page_size=5 HTTP/1.1" 200 12114 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.13"
2021-05-13 17:00:34,794 INFO     [2abfdaf0a4af4a72b789829e3cd2201b] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/unified_job_templates/'>, response_time: 0.372s
[pid: 27|app: 0|req: 417/2898] 10.43.237.14 () {74 vars in 3359 bytes} [Thu May 13 17:00:34 2021] OPTIONS /api/v2/unified_job_templates/ => generated 9598 bytes in 373 msecs (HTTP/1.1 200) 14 headers in 571 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:34 +0000] "OPTIONS /api/v2/unified_job_templates/ HTTP/1.1" 200 9598 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.14"
2021-05-13 17:00:36,828 INFO     [93f31d945dae45988bf57bee26c45efe] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_job_templates/?order_by=name&page=1&page_size=20&type=job_template%2Cworkflow_job_template'>, response_time: 2.838s
[pid: 29|app: 0|req: 825/2899] 10.43.237.14 () {72 vars in 3461 bytes} [Thu May 13 17:00:33 2021] GET /api/v2/unified_job_templates/?order_by=name&page=1&page_size=20&type=job_template%2Cworkflow_job_template => generated 81912 bytes in 2841 msecs (HTTP/1.1 200) 14 headers in 572 bytes (1 switches on core 0)
10.43.237.14 - - [13/May/2021:17:00:36 +0000] "GET /api/v2/unified_job_templates/?order_by=name&page=1&page_size=20&type=job_template%2Cworkflow_job_template HTTP/1.1" 200 81912 "https://awx.<redacted>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "10.145.194.13"
```

**Comment by AlexSCorey** [comment]
@dacosta89 Check out https://github.com/ansible/awx/pull/10186 and see if that solves your problem.

**Comment by jakemcdermott** [comment]
I believe this should be fixed by https://github.com/ansible/awx/pull/10186 but we'll confirm

**Comment by keithjgrant** [comment]
Confirmed this is resolved by #10215
